### ISSUE TITLE
Add MISP attribute comment as STIX indicator description 

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#323]
Currently the MISP attribute comments are lost when exporting events to STIX. Please consider this patch that preserves them in the STIX indicator description field.
```
diff --git a/app/files/scripts/misp2stix.py b/app/files/scripts/misp2stix.py
index e6dacb3..b83adae 100644
--- a/app/files/scripts/misp2stix.py
+++ b/app/files/scripts/misp2stix.py
@@ -256,6 +256,8 @@ def generateThreatActor(attribute):
 def generateIndicator(attribute):
     indicator = Indicator()
     indicator.id_="example:indicator-" + attribute["uuid"]
+    if attribute["comment"] != "":
+        indicator.description = attribute["comment"]
     setTLP(indicator, attribute["distribution"])
     indicator.title = "MISP Attribute #" + attribute["id"] + " uuid: " + attribute["uuid"]
     confidence_description = "Derived from MISP's IDS flag. If an attribute is marked for IDS exports, the confidence will be high, otherwise none"
```

Btw, indicator.title is set twice to the same value, first in generateIndicator() and then again in handleIndicatorAttribute().

**Comment by RichieB2B** [comment]
I think I have mapped all possible comments into STIX, except for the victim Identity which is created with resolveIdentityAttribute() in misp2ciq.py. The STIX/CIQ datamodel doesn't seem to have an appriate field in this case.

**Comment by RichieB2B** [comment]
Fixed in c3615ef
